test/interface/test_filters.py

```diff
@@ -179,6 +179,37 @@ def test_multiple_models(self, session):
         assert {qux.id for qux in quxs} == {1, 3}
         assert {qux.name for qux in quxs} == {"name_1"}
 
+    @pytest.mark.usefixtures('multiple_bars_inserted')
+    @pytest.mark.usefixtures('multiple_quxs_inserted')
+    def test_multiple_model_hybrid_fields(self, session):
+        query = session.query(Bar, Qux)
+        filters = [
+            {
+                'model': 'Bar',
+                'field': 'count_square',
+                'op': '==',
+                'value': 100
+            },
+            {
+                'model': 'Qux',
+                'field': 'count_square',
+                'op': '>=',
+                'value': 25
+            },
+        ]
+
+        filtered_query = apply_filters(query, filters)
+        result = filtered_query.all()
+
+        assert len(result) == 3
+        bars, quxs = zip(*result)
+        assert set(map(type, bars)) == {Bar}
+        assert {bar.id for bar in bars} == {2}
```

Comment:
We can do `assert [bar.count for bar in bars] == [10]` instead since it's easier to understand without having to look at the pytest fixture to check the `count` value for the `Bar` element with ID `2`.